### ISSUE TITLE
Map PinView to a button

### ISSUE THREAD

**Issue by developer9969** [MGohil/PINView.Maui#3]
Hi
Wondering how I could use your control again a set of buttons in my page.
I Have a custom "Keypad page" with 10 buttons - 1 to 9 buttons and 0 button and when I press a button i want to trigger your pinview control and capture the value . 

Does your pinView only work with the inbuilt keyboard or Can I somehow map the PinValue to the value of a button that gets pressed on my page?

Does it make sense what I am asking?

many thanks

**Comment by MGohil** [comment]
@developer9969 : Yes, I got your point. Currently this control only supports inbuilt keybaord. There is a hidden textbox behind the boxes which takes input keys from soft keyboard.

I liked the way you are thinking of having this control supporting custom numeric buttons on screen, but I am not sure when I can make it possible.

Thanks.
